ktlint/src/main/kotlin/com/pinterest/ktlint/Main.kt

```diff
@@ -132,6 +133,12 @@ class KtlintCommandLine {
     )
     var color: Boolean = false
 
+    @Option(
+        names = ["--color-name"],
+        description = ["Customize the output color"]
+    )
+    var colorName: String = Color.DARK_GRAY.name
```

Comment:
Would be nice to use here enum type, but it will require a lot of changes :(

Reply:
I agree and thought the same but I wasn't sure if I could reference an enum as a command line argument? I assumed it had to be a string. If you know something I don't - let me know and I can try to adjust.